moped-editor/src/views/dashboard/DashboardView.js

```diff
@@ -231,6 +233,7 @@ const DashboardView = () => {
         <DashboardStatusModal
           projectId={entry.project_id}
           projectName={entry.project.project_name}
+          currentPhaseId={entry.current_phase_id}
```

Comment:
fixes an bug where the note's `phase_id` was not being set from the dashboard status modal